### ISSUE TITLE
Maintain current app version centrally

### ISSUE THREAD

**Issue by tkowark** [corona-warn-app/cwa-website#292]
## Feature description

The current app version should be maintained only once. The feature would introduce one point where the version is set and then it gets properly set in every FAQ, page text, etc.

Top of mind proposal: Introduce a search and replace task in the gulpfile that parses each html, json, and md file for a certain string (e.g., $$CWA_CUR_ANDROID$$) and replaces it with the value of an environment variable or value read from a file.

## Problem and motivation

App versions are spread all across the page. We should neither have to track them manually, nor touch multiple files when it changes.

## Is this something you're interested in working on

Yes

**Comment by kereng5** [comment]
https://www.coronawarn.app says under "This is how the app works best": "Download the current version of the app (**Android: 1.2.0** or iOS: 1.2.1)".
Same in German: Aktuelle Version der App laden (**Android: 1.2.0** bzw. iOS: 1.2.1)

**Comment by tkowark** [comment]
Thanks for spotting that. Fixed with https://github.com/corona-warn-app/cwa-website/commit/6f32bea527a4a311a32136cb3dae01fff848a7aa ... and that's exactly the reason why this needs to be stored only once

**Comment by MikeMcC399** [comment]
@tkowark 
The issue raised by @kereng5 has occurred again as the current version has moved to 1.3.0
https://www.coronawarn.app/en/ says "Download the current version of the app (Android: 1.2.1 or iOS: 1.2.1)"

If the current app version is not maintained centrally then at least the release process needs to ensure that the current version number used in public places is updated manually when a new version is released.

**Comment by Ein-Tim** [comment]
@MikeMcC399 
I've already opend a Pr for that:

https://github.com/corona-warn-app/cwa-website/pull/371

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> I've already opend a PR for that: #371 

Thanks for catching and fixing!

**Comment by Ein-Tim** [comment]
This will be introduced with #420

**Comment by MikeMcC399** [comment]
I think this is complete, even though #460 has an "in progress" label attached to it. 1.5.0 was released yesterday, Oct 19, 2020.

**Comment by mtb77** [comment]
@svengabr isn't this already fixed with your latest update?

**Comment by svengabr** [comment]
Yes, I did already implement that.

You can now use placeholders in the JSON Files like follows:
```[ios.latest-os-version]
[ios.minimum-required-os-version]
[ios.current-app-version]
[android.latest-os-version]
[android.minimum-required-os-version]
[android.current-app-version]
[last-update]
```

**Comment by kereng5** [comment]
This issue is "fixed" and "closed", but on coronawarn.app/de I still read: 
"Aktuelle Version der App laden (iOS: 1.5.3 bzw. Android: 1.5.1)"
while 1.6.0 is installed.

**Comment by Ein-Tim** [comment]
@kereng5 See #559

**Comment by MikeMcC399** [comment]
@kereng5 / @Ein-Tim 
I raised this last time with @svengabr who asked me via Slack to raise a PR, which I did in https://github.com/corona-warn-app/cwa-website/pull/517. 
I don't think that somebody from the community should be maintaining the variables:

```
[ios.current-app-version]
[android.current-app-version]
```
in https://github.com/corona-warn-app/cwa-website/blob/master/gulpfile.js. The Open Source Team needs to take ownership of maintaining these variables in a timely fashion.

So this time I didn't raise a PR, but instead I opened the Issue #559 which @Ein-Tim referred to.
I hope that somebody from the Open Source Team will respond and take action soon.
